bucket/better-joy.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "6.4b",
+    "description": "Allows the Nintendo Switch Pro Controller, Joycons and SNES controller to be used with CEMU, Citra, Dolphin, Yuzu and as generic XInput",
+    "homepage": "https://github.com/Davidobot/BetterJoy",
+    "license": "MIT",
+    "notes": "How to use: https://github.com/Davidobot/BetterJoy#how-to-use",
```

Comment:
Irrelevant

```suggestion
```

Reply:
For BetterJoy to work properly, wo must to install a driver manually. So I added a link to README...

Reply:
I have modified `notes` to `Please install the drivers in the folder: $dir\\Drivers`.